src/install.rst

```diff
@@ -6,148 +6,184 @@ Installing Nextstrain
 
-        conda activate nextstrain
-        nextstrain check-setup --set-default
+         - This is necessary for a dependency of Nextstrain CLI and `we are investigating options to eliminate this step <https://github.com/nextstrain/cli/issues/31#issuecomment-970641263>`_.
 
-    The final output from the last command should look like this:
+      3. Open an Anaconda Prompt, which can be found in the Start menu.
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        Setting default environment to docker.
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-    Finally, download the latest Docker image for Nextstrain.
+      5. Install the Nextstrain build/view environment by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+      6. Confirm that the installation worked.
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        nextstrain update
+            nextstrain check-setup --set-default
 
-    Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-    Next, try :doc:`tutorials/quickstart`.
+         The final output from the last command should look like this:
 
-Upgrade an existing installation
-================================
+         .. code-block:: none
 
-.. tabs::
+            Setting default environment to docker.
+
+   .. group-tab:: WSL on Windows
+
+      .. note::
+
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
+
+      1. `Install WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+      2. Open a WSL terminal by running **wsl** from the Start menu.
+      3. Install Miniconda:
+
+         .. code-block:: bash
+
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh
+            rm Miniconda3-latest-Linux-x86_64.sh
 
-   .. group-tab:: Conda
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-      Update the base Conda environment.
+         .. code-block:: bash
 
-      .. code-block:: bash
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-         conda update -n base conda
+      5. Install the Nextstrain build/view environment. There are two options:
 
-      Update the Nextstrain environment.
+         a. Docker (recommended) – on Windows, `install Docker Desktop for WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
```

Comment:
I hadn't thought about this combination of possible installation options and I haven't heard of anyone using it before.

Is it worth keeping a separate "WSL for Windows" option, if the "Windows" option does everything we need it to? I guess the main argument for this option is to provide a way for users to install with Conda (and avoid Docker). @emmahodcroft, do you have any opinions about this?

Either way, these are great instructions for a "Linux" tab (minus the Windows-specific bits).

Reply:
I agree, Windows+Docker should be sufficient for most users. But if a Windows user wants to install a specific version of augur/auspice or do any development work, WSL is the only option. For now, we can keep the tab here with the option to move it elsewhere later?

Reply:
That sounds good!